client/src/pages/Profile.js

```diff
@@ -25,18 +22,12 @@ const Profile = () => {
                     </div>
                     <div className="follow-container">
                         <ul>
-                            <NavLink to="/dashboard/profile/follow" activeClassName="current"><li>Follow</li></NavLink>
-                            <NavLink to="/dashboard/profile/message" activeClassName="current"><li>Message</li></NavLink>
+                            <NavLink to="/profile/follow" activeClassName="current"><li>Follow</li></NavLink>
```

Comment:
These shouldn't be NavLinks. They should be regular buttons that will have an action in the future. So they shouldn't take the user anywhere